arkouda/array_api/_typing.py

```diff
@@ -44,11 +44,9 @@
 
 
 class NestedSequence(Protocol[_T_co]):
-    def __getitem__(self, key: int, /) -> _T_co | NestedSequence[_T_co]:
-        ...
+    def __getitem__(self, key: int, /) -> _T_co | NestedSequence[_T_co]: ...
 
-    def __len__(self, /) -> int:
-        ...
+    def __len__(self, /) -> int: ...
```

Comment:
Why were these `...`s added?

Reply:
I think the ellipses just got moved up from the next line by the auto-formatter I used.